circle.yml

```diff
@@ -34,7 +34,7 @@ checkout:
 
 test:
   override:
-    - case $CIRCLE_NODE_INDEX in 0) (cd app; meteor npm run test) ;; 1) (cd app; meteor npm run test-app) ;; 2) (cd app; ./.testing/install_chimp.sh; ./tests/acceptance_run) ;; esac:
+    - case $CIRCLE_NODE_INDEX in 0) (cd app; meteor npm run test) ;; 1) (cd app; meteor npm run coverage) ;; 2) (cd app; ./.testing/install_chimp.sh; ./tests/acceptance_run) ;; esac:
```

Comment:
Totally, it actually merge the coverage reports: 
- the first command is coverage-app-unit : spacejam-mocha [..] --coverage '[..] out_coverage', it exports the coverage dump
- the second command is coverage-app-full: spacejam-mocha [..] --coverage '[..] in_coverage', it imports the coverage dump before exporting anything

I don't understand the parallelization of circle-ci, so I can't help. Just keep in mind that coverage-app-unit needs to export the dump to let coverage-app-full succeed.